ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -428,19 +428,14 @@ objects:
         wait "$!"
 
         function approve_csrs() {
-          while true; do
-            if [[ ! -f /tmp/install-complete ]]; then
-              oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve
-              sleep 15 & wait
-              continue
-            else
-              break
-            fi
+          while [[ ! -f /tmp/exit ]] || [[ ! -f /tmp/setup-success ]]; do
```

Comment:
Needs to be [`&&`][1].

[1]: https://github.com/openshift/release/pull/3587/files#diff-1c5ad26c50e4b0874d0c277f2152d68fR303